src/main/java/core/model/HOVerwaltung.java

```diff
@@ -90,9 +86,13 @@ public void setResource(String pfad) {
 		setTranslator(pfad);
 	}
 
+	/**
+	 * @deprecated Provided for compatibility. Please use {@link TranslationFacility#setLanguage(String)} instead.
+	 */
+	@Deprecated(since = "9.0", forRemoval = true)
```

Comment:
Also possible but it might be difficult to merge from release branches when some refactoring is already in the master branch.

Reply:
Not sure I get this.  `master` will be for a `9.0`, which is where the refactoring is ongoing, when the release branches would theoretically only get bug fixes.  Why would there be issues merging?  AFAIU this specific `setTranslator` method was introduced on `master` and `9.0` was not released so it should be fine to remove?

Reply:
> Not sure I get this. `master` will be for a `9.0`, which is where the refactoring is ongoing, when the release branches would theoretically only get bug fixes. Why would there be issues merging? AFAIU this specific `setTranslator` method was introduced on `master` and `9.0` was not released so it should be fine to remove?

It depends on the branching model we are using.
What I saw is:
There is a release branch `release/8` to make the application stable.
The release branch is a branch from `master`.

Bugs are fixed in that branch `release/8` to get a usable product.
Then the release it built from this release branch.
Right?

The main question is **how** the fixes/changes in that release branch are merged into the `master` branch.
The master branch needs these changes normally because the `master` branch is the origin for next release branch.
When the master branch is not updated with these fixes/changes the bugs occur again when a release is created from `master` for the next release branch.

The normal GIT action to perform that update from the `master` branch is to merge from the current release branch that contains the fixes/changes.

**In the normal development process there are these refactoring like the PR here.**

So when you think of a fix of a line in the release branch that used the old API before this PR here (when resident in master) was merged, you will get compile time errors.
To prevent this you can still use the old and deprecated API here.
After the last release of version `9`, so when the next release branch `release/9` will not get any update, the deprecated API can be removed in master.

So basically the idea behind the annotation for deprecation is to help merging from release branches to the master branch.
This does not help for merge conflicts but if there are any you have the advantage to use parts of the old API here.

BTW:
I see no merge lines in the GIT tree.

For example here:
#2097 was fixed with 5c50df018408b1c4c6e41512fe5322faa2849159 in `release/8` branch
The fix was related to file 'MatchLineupDetails.java'
With #2100 and commit 51049cd0c262c8dbbe960b4649638b5af9a351fe the same thing was done in the `master` branch be you cannot see that in the GIT tree.

How are you creating these merges?

Reply:
Usually @wsbrenk takes care of the release branching, so I'll let him pitch in here.  But IMHO, this is making things more complicated than they need to be.  If you're merging, and you're getting conflicts, you fix at merge time.  I cannot recall a situation where this happened in HO, but @wsbrenk is probably better placed to comment.

Reply:
Right, I think I'll let it go to move ahead with that PR that's be open for a while, but let's reconsider that pattern.